README.rst

```diff
@@ -1,86 +1,91 @@
+undergoing considerable redevelopment. The aim is to have a small and simple
+and performant library for reading and writing the parquet format from python.
 
-getting started
-===============
+.. _parquet-python: https://github.com/jcrobak/parquet-python
 
-parquet-python is available via PyPi and can be installed using
-`pip install parquet`. The package includes the `parquet`
-command for reading python files, e.g. `parquet test.parquet`.
-See `parquet --help` for full usage.
+A list of expected features and their status in this branch can be found in
+`this issue`_.
+Please feel free to comment on that list as to missing items and priorities.
 
-Example
--------
+.. _this issue: https://github.com/martindurant/fastparquet/issues/1
 
-parquet-python currently has two programatic interfaces with similar
-functionality to Python's csv reader. First, it supports a DictReader
-which returns a dictionary per row. Second, it has a reader which
-returns a list of values for each row. Both function require a file-like
-object and support an optional ``columns`` field to only read the
-specified columns.
+In the meantime, the more eyes on this code, the more example files and the
+more use cases the better.
 
-.. code:: python
+For the time being, this code should be considered extreme alpha.
 
+Requirements
+------------
 
-    import parquet
-    import json
+(all development is against recent versions in the default anaconda channels)
 
-    ## assuming parquet file with two rows and three columns:
-    ## foo bar baz
-    ## 1   2   3
-    ## 4   5   6
+Required:
 
-    with open("test.parquet") as fo:
-       # prints:
-       # {"foo": 1, "bar": 2}
-       # {"foo": 4, "bar": 5}
-       for row in parquet.DictReader(fo, columns=['foo', 'bar']):
-           print(json.dumps(row))
+- numba
+- numpy
+- pandas
 
+Optional (compression algorithms; gzip is always available):
 
-    with open("test.parquet") as fo:
-       # prints:
-       # 1,2
-       # 4,5
-       for row in parquet.reader(fo, columns=['foo', 'bar]):
-           print(",".join([str(r) for r in row]))
+- snappy
+- lzo
+- brotli
 
-Todos
-=====
+Installation
+------------
 
--  Support the deprecated bitpacking
--  Fix handling of repetition-levels and definition-levels
--  Tests for nested schemas, null data
--  Support reading of data from HDFS via snakebite and/or webhdfs.
--  Implement writing
--  performance evaluation and optimization (i.e. how does it compare to
-   the c++, java implementations)
+    > pip install git+https://github.com/martindurant/fastparquet
 
-Contributing
-============
+Assuming the requirements have been met. Numba should be installed using conda,
+and a conda package of this package will be forthcoming.
 
-Is done via Pull Requests. Please include tests with your changes and
-follow `pep8 <http://www.python.org/dev/peps/pep-0008/>`_.
+Usage
+-----
+
+*Reading*
+
+.. code-block:: python
+
+    import fastparquet
```

Comment:
Recommend using `from fastparquet import ParquetFile` in future examples so that if we need to change the name we'll have to rename less